File: server/filmpalace/src/main/java/com/rossetti/filmpalace/dto/EditOutput.java
package com.rossetti.filmpalace.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotNull;
import java.util.List;

@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class EditOutput {
    @NotNull
    private List<GeneroOutput> generos;
    @NotNull
    private List<AtorOutput> atores;
    @NotNull
    private List<CinemaOutput> cinemas;
}

File: web/src/app/filmes/services/models/edit-output.ts
import {GeneroOutput} from "../../../generos/services/models/genero-output";
import {AtorOutput} from "../../../atores/services/models/ator-output";
import {CinemaOutput} from "../../../cinemas/services/models/cinema-output";

export interface EditOutput {
  generos: GeneroOutput[];
  atores: AtorOutput[];
  cinemas: CinemaOutput[];
}
